home/tests/import-export-data/whatsapp-template-simple.csv

```diff
@@ -1,2 +1,2 @@
-name,category,quick_replies,locale,image,message,example_values,submission_name,submission_status,submission_result
-Test Template Simple,MARKETING,,en,,Message,,,,
+name,category,buttons,locale,image,message,example_values,submission_name,submission_status,submission_result
+Test Template Simple,MARKETING,"[{""type"": ""next_message"", ""title"": ""Sure, remind me"", ""slug"": """"}, {""type"": ""next_message"", ""title"": ""No, go to main menu"", ""slug"": """"}]",en,,Message,,,,
```

Comment:
Maybe one of these buttons should be a `go_to_page`, and one a `go_to_form`, so that we can test all the button types for import?